galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/ITsqHandler.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.cicsts;
+
+import javax.validation.constraints.NotNull;
+
+public interface ITsqHandler {
```

Comment:
If this interface represents a single specific TSQ, calling it ITsq or ITSQ might make more sense ?

Reply:
Hi @techcobweb , I had a discussion with @Jimbo4794  on this. He suggested to use ITsqHandler instead of ITsq as this is an instance used to handle TSQs. This is not creating a TSQ, instead it is providing a handler to perform various actions on TSQs like read, write (create), delete, make recoverable etc. This name is similar to ZosFileHandler.  Happy to discuss further on this to finalize the name.